### ISSUE TITLE
Documentation "deferredindepth.html" incorrect output

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2582]
|<img alt="jaimecham's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| jaimecham reported|
|-|-|
|Trac ID|trac#2582|
|Type|defect|
|Created|2007-04-20 06:40:47Z|

The output of the section "Handling an exception and continuing on" is wrong. Currently it's:
```
callback 1
        got result: success
        about to raise exception
errback
we got an exception: Traceback (most recent call last):
  File "./deferred_ex2.py", line 85, in ?
    nonDeferredExample("success")
--- <exception caught here> ---
  File "./deferred_ex2.py", line 46, in nonDeferredExample
    result = failAtHandlingResult(result)
  File "./deferred_ex2.py", line 35, in failAtHandlingResult
    raise RuntimeError, "whoops! we encountered an error"
exceptions.RuntimeError: whoops! we encountered an error
```
Should be something like:
```
callback 1
 got result: success
callback 2
 got result: yay! handleResult was successful!
 about to raise exception
errback
we got an exception: Traceback (most recent call last):
  File "D:\jcham\Eclipse-Workspace\Graffiki-02\Server\twisted_samplecode\deferred_ex3.py", line 97, in <module>
    deferredExample()
  File "D:\jcham\Eclipse-Workspace\Graffiki-02\Server\twisted_samplecode\deferred_ex3.py", line 90, in deferredExample
    d.callback("success")
  File "C:\Python25\lib\site-packages\twisted\internet\defer.py", line 239, in callback
    self._startRunCallbacks(result)
  File "C:\Python25\lib\site-packages\twisted\internet\defer.py", line 304, in _startRunCallbacks
    self._runCallbacks()
--- <exception caught here> ---
  File "C:\Python25\lib\site-packages\twisted\internet\defer.py", line 317, in _runCallbacks
    self.result = callback(self.result, *args, **kw)
  File "D:\jcham\Eclipse-Workspace\Graffiki-02\Server\twisted_samplecode\deferred_ex3.py", line 34, in failAtHandlingResult
    raise RuntimeError, "whoops! we encountered an error"
exceptions.RuntimeError: whoops! we encountered an error

callback 3
 got result: okay, continue on
```

Attachments:

* [deferredindepth-doc-2582.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/541/541e1ca1ba795f8e006cdb4392dae898e5678fda/3c9c57f1e0cf8454b5ef675ec9ff59b5113ea59a.patch) (1551 bytes) - added by thijs on 2008-07-14 07:16:44Z - patch against r24269

<details><summary>Searchable metadata</summary>

```
trac-id__2582 2582
type__defect defect
reporter__jaimecham jaimecham
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1177051247000000 1177051247000000
changetime__1216304544000000 1216304544000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

See attached [[deferredindepth-doc-2582.patch](None/deferredindepth-doc-2582.patch) patch] for a fix.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set owner to @thijstriemstra|
|-|-|

Great, please apply.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24282]) Apply deferredindepth-doc-2582.patch: The output of the section "Handling an exception and continuing on" was incorrect.

Author: jaimecham, thijs
Reviewer: therve
Fixes: [#2582](https://github.com/twisted/twisted/issues/2582)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
